models/agent.go

```diff
@@ -182,3 +184,79 @@ type RDSExporter struct {
 
 	ListenPort *uint16 `reform:"listen_port"`
 }
+
+// AgentFilters represents filters for agents list.
+type AgentFilters struct {
+	// Return only Agents running on that Node.
+	HostNodeID string
```

Comment:
Please rename it to `RunsOnNodeID`. `Host` concept will be removed, API changes are coming.

Reply:
Another important change: let's allow only a single field (filter) to be set. Return gRPC error if two or more filters are set. We may relax that requirement in the future, but the complex filters do not make sense for the current UX plans.